README.md

```diff
@@ -57,3 +57,17 @@ API docs are at http://www.ev3dev.org/ev3devKit/ev3devkit/index.htm
 [GObjectIntrospection]: https://wiki.gnome.org/Projects/GObjectIntrospection
 [brickman]: https://github.com/ev3dev/brickman
 [brickstrap]: https://github.com/ev3dev/ev3dev/wiki/Using-brickstrap-to-cross-compile-and-debug
+
+## Running
+
+The demos can be run on the desktop from the build dir like below:
+The vala ui demo required the package was built with `-DEV3DEVKIT_DESKTOP=bool:Yes`
+
+    ./ev3devKit-ui-demo
```

Comment:
There are actually run targets in the make file. Type `make run<tab>` and you will see the demos. IIRC, it will set the proper environment for you instead of typing all of the stuff below. However, it is good to have the example below for running on the EV3 itself if you manually copy the files.